<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TheaterAddModifyForm</class>
 <widget class="QDialog" name="TheaterAddModifyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>894</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MyResources.qrc">
    <normaloff>:/Images/theater pic.png</normaloff>:/Images/theater pic.png</iconset>
  </property>
  <widget class="QLabel" name="lbllblId">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Id</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblId">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbllblId_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbllblId_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Theater type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxTheaterType">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtSave">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtCancel">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtAddSeat">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Seat</string>
   </property>
  </widget>
  <widget class="QWidget" name="widTheater" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>1201</width>
     <height>800</height>
    </rect>
   </property>
   <widget class="QLabel" name="lblTheater">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="MyResources.qrc">:/Images/movie theater.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Seat row</string>
   </property>
  </widget>
  <widget class="CustomLineEdit" name="ledSeatRow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Seat column</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbSeatColumn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtModifyRowColumn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Modify 
row/column</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtAddHandicapSeat">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>30</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Handicap 
Seat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtRemoveSeat">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>remove seat</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">customlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MyResources.qrc"/>
 </resources>
 <connections/>
</ui>
